.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
probably best not to add this file without something to enforce its correctness? (eslint doesn't check editorconfig)

Reply:
![Screen Shot 2022-07-15 at 1 55 42 PM](https://user-images.githubusercontent.com/15326526/179292556-2a95bd00-6551-40a3-94a6-9619d98b86c8.png)

Reply:
interesting, thanks. it didn't really click for me that this was using super-linter :-/ but i guess we'll stick with it